content/docs/get-started/index.md

```diff
@@ -250,13 +208,22 @@ $ mlem apply models/rf new_data.csv \
     --method predict_proba \
     --import \
     --import-type "pandas[csv]"
+```
```

Comment:
we're leaving out `--batch_size 1` to highlight "batchness"?

Reply:
since it's a toy example batches don't make much difference. Would you like me to add the flag to show it exists? I'm not sure it adds much. the "batch" operation here, I think, it demonstrated by showing it's a dataset we're applying and not a vector